### ISSUE TITLE
Add light failed in blender 3.1

### ISSUE THREAD

**Issue by atticus-lv** [leomoon-studios/blender-light-studio#62]
```Python: Traceback (most recent call last):
  File "C:\Users\atticus\AppData\Roaming\Blender Foundation\Blender\3.1\scripts\addons\leomoon-lightstudio-master\light_operators.py", line 331, in execute
    advanced_light_collection = [c for c in collection.children if c.name.startswith('LLS_Advanced')][0]
IndexError: list index out of range

location: <unknown location>:-1

```

![image](https://user-images.githubusercontent.com/64118610/148761372-725d814a-0b5d-4804-b3d7-c718f9e32564.png)

**Comment by leomoon** [comment]
Hi,

Blender 3.1 is not final. We will fix it once the final version is out.

**Comment by atticus-lv** [comment]
I add some log here
![image](https://user-images.githubusercontent.com/64118610/149001821-38d146ae-5b14-4c1b-a35d-cd88acda8697.png)

And get a result like this
![image](https://user-images.githubusercontent.com/64118610/149001792-220589ba-5cee-4143-8b72-b90c23ff045e.png)
It is just like the bug I report to blender before.  I find it in my own addon by appending node group in a same way
https://developer.blender.org/T94338

**Comment by leomoon** [comment]
perfect. hopefully they will fix this before the final version is out.

Thanks for the detailed info. ;)

**Comment by atticus-lv** [comment]
Bug fix! Addon now can be use in 3.1
